.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
If you had issues enabled, I could say that this PR is related to an issue, instead I'll link it back to a PR #1